docs/docs/en/development/development-environment-setup.md

```diff
@@ -23,6 +23,24 @@ git clone git@github.com:apache/dolphinscheduler.git
 
 ii. Run `mvn clean install -Prelease -Dmaven.test.skip=true`
 
+### Docker image build
+* Run
+```shell
+$ cd dolphinscheduler
+$ ./mvnw -B clean deploy \
+          -Dmaven.test.skip \
+          -Dmaven.javadoc.skip \
+          -Dmaven.checkstyle.skip \
+          -Dmaven.deploy.skip \
+          -Ddocker.tag=<TAG> \
+          -Ddocker.hub=<HUB_URL> \
+          -Pdocker,release          
+```
```

Comment:
* To build Docker images locally, run

```shell
$ cd dolphinscheduler
$ ./mvnw -B clean package \
          -Dmaven.test.skip \
          -Dmaven.javadoc.skip \
          -Dmaven.checkstyle.skip \
          -Ddocker.tag=<TAG> \
          -Pdocker,release          
```

* To build and push Docker images to your registry `<HUB_URL>`, run
```shell
$ cd dolphinscheduler
$ ./mvnw -B clean deploy \
          -Dmaven.test.skip \
          -Dmaven.javadoc.skip \
          -Dmaven.checkstyle.skip \
          -Dmaven.deploy.skip \
          -Ddocker.tag=<TAG> \
          -Ddocker.hub=<HUB_URL> \
          -Pdocker,release          
```

Reply:
@liqingwang please update the doc to 👆

Reply:
Done